addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -22,23 +22,42 @@
                         </div>
                     </div>
                 </section>
-                <div t-if="event.is_finished" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is finished. It's no longer possible to book a booth.</span>
-                </div>
-                <div t-elif="not event_booths" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is not open to exhibitors registration,
-                        <span t-if="request.env.user.has_group('event.group_event_manager')">
-                            you can
-                            <a class="text-nowrap" t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
-                                <i class="fa fa-gear me-1" role="img" aria-label="Configure" title="Configure event booths"/><em>Configure Booths</em>
-                            </a>
-                            for this event.
-                        </span>
-                        <span t-else="">
-                            check our <a href="/event">list of future events</a>.
-                        </span>
-                    </span>
-                </div>
+                <t t-if="event.is_finished">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-12 text-center">
+                                <div class="my-4">
+                                    <t t-call="website_event.event_svg"/>
+                                </div>
+                                <h2>Event Finished</h2>
+                                <p>It's no longer possible to book a booth.</p>
+                            </div>
+                        </div>
+                    </div>
+                </t>
+                <t t-elif="not event_booths">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-12 text-center">
+                                <div class="my-4">
+                                    <t t-call="website_event.event_svg"/>
+                                </div>
+                                <h2>Registration Not Open</h2>
+                                <p>This event is not open to exhibitors registration at this time</p>
+                                <div class="o_not_editable my-3" groups="event.group_event_user">
+                                    <div t-if="request.env.user.has_group('event.group_event_manager')">
```

Comment:
One of these group checks should not be there. I would use `groups="event.group_event_manager"` as only managers are allowed to create booths. I couldn't find any justification for any of the few places that use `has_group()` instead of `groups=`, and this seems to work and be used elsewhere in the template so it should be fine to go with either. Up to you.

(I'm partial to just using `groups=` everywhere though)